### ISSUE TITLE
Md.Parser is not available

### ISSUE THREAD

**Issue by ambareesha7** [derek-zhou/gara#1]
Hey @derek-zhou thank you for your work.

there is no md folder included in GitHub repo that's I'm getting this error

```
== Compilation error in file lib/gara/parser.ex ==
** (CompileError) lib/gara/parser.ex:2: module Md.Parser is not loaded and could not be found
    (elixir 1.13.4) expanding macro: Kernel.use/1
    lib/gara/parser.ex:2: Gara.Parser (module)
```
Please suggest me how to handle this

**Comment by derek-zhou** [comment]
Md is a new dependency, replacing the venerable Earmark for more flexible markdown syntax. did you do a `mix deps.get`?

**Comment by derek-zhou** [comment]
Sorry, my bad. I did not `git push` 

Fixed in 9bff12a

**Comment by ambareesha7** [comment]
hey thank you man its working
